fdbserver/worker.actor.cpp

```diff
@@ -2497,11 +2514,20 @@ ACTOR Future<Void> workerServer(Reference<IClusterConnectionRecord> connRecord,
 				loggingTrigger = delay(loggingDelay, TaskPriority::FlushTrace);
 			}
 			when(state WorkerSnapRequest snapReq = waitNext(interf.workerSnapReq.getFuture())) {
-				Standalone<StringRef> snapFolder = StringRef(folder);
-				if (snapReq.role.toString() == "coord") {
-					snapFolder = coordFolder;
+				if (g_network->isSimulated() && lastSnapReq.snapUID == snapReq.snapUID &&
```

Comment:
If one process is snapshotted more than once in one snapshot round(the UID), we report the error.
Maybe we can remove the gap time check `(now() - lastSnapTime) < SERVER_KNOBS->SNAP_CREATE_MAX_TIMEOUT)` as two different snapshots are always having different UIDs